CHANGELOG.md

```diff
@@ -1,8 +1,17 @@
 
 # [unreleased]
-
 ## New features
 
+- New syntax for memory load/store (uXX)[aligned? x e] become [aligned? :uXX x e],
+  the syntax  (uXX)[aligned? x e] is deprecated.
+  New syntax for array load/store t DOT? [aligned? uXX e] become t DOT? [aligned? :uXX e],
+  the syntax t DOT? [aligned? uXX e] is deprecated.
+  The script compiler/script/replace.sh allows to perform the change automatically.
```

Comment:
Just update the name of the script here if you change its name.

Reply:
done. It is now replace-new-memory-syntax.sh

Reply:
Is it ok ?

Reply:
For me, it is.